core/src/test/java/org/apache/spark/launcher/SparkLauncherSuite.java

```diff
@@ -133,6 +134,10 @@ public void testInProcessLauncher() throws Exception {
         p.put(e.getKey(), e.getValue());
       }
       System.setProperties(p);
+      // Here DAGScheduler is stopped, while SparkContext.clearActiveContext may not be called yet.
+      // Wait for a reasonable amount of time to avoid creating two active SparkContext in JVM.
+      // See SPARK-23019 and SparkContext.stop() for details.
+      TimeUnit.MILLISECONDS.sleep(500);
```

Comment:
I usually just see people write `Thread.sleep(500)` but no big deal.
This is somewhat hacky, but may be OK for just this test's purpose.

Reply:
Yep, either way should work.
Using `TimeUnit` is just following `BaseSuite.java`.